app/outputs/HarvestApp/Output.ts

```diff
@@ -19,6 +19,7 @@ export class HarvestAppOutput extends OutputBase {
 
         this.harvestClient = new HarvestClient(outputConfiguration);
         this.configuration = outputConfiguration;
+    this.name = this.outputConfiguration.name;
```

Comment:
This line is incorrectly indented.

```suggestion
        this.name = this.outputConfiguration.name;
```